packages/plugin-commands-store/src/store.ts

```diff
@@ -94,6 +96,10 @@ export async function handler (opts: StoreCommandOptions, params: string[]) {
   }
 }
 
+async function pathCmd (opts: StoreCommandOptions) {
+  return await storePath(opts.dir, opts.storeDir)
+}
```

Comment:
why do you need this function at all? just run storePath at line 76

Reply:
Updated 👍